src/job_cache/eviction_policy.cpp

```diff
@@ -80,8 +84,166 @@ struct CommandParser {
   }
 };
 
-int eviction_loop(std::unique_ptr<EvictionPolicy> policy) {
-  policy->init();
+struct LRUEvictionPolicyImpl {
+  std::string cache_dir;
+  job_cache::PreparedStatement update_size;
+  job_cache::PreparedStatement get_size;
+  job_cache::PreparedStatement update_last_use;
+  job_cache::PreparedStatement find_least_recently_used;
+  job_cache::PreparedStatement remove_least_recently_used;
+  job_cache::Transaction transact;
+
+  // TODO: Right now we're using `obytes` as a proxy for the size of a job
+  //       but this is an over aproximation of the storage cost really.
+
```

Comment:
mentioned in DM, but we probably want a "resync/recalc total size" query that is called on some interval